miio/curtain_babai.py

```diff
@@ -0,0 +1,201 @@
+from enum import Enum
+from time import sleep
+from typing import Any, Dict, List
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .exceptions import DeviceError
+from .miot_device import DeviceStatus, MiotDevice
+
+_MAPPING = {
+    # Source
+    # http://miot-spec.org/miot-spec-v2/instances?status=all
+    # https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:curtain:0000A00C:babai-190812:1:0000C805
+    # Curtain
+    "motor_control": {"siid": 2, "piid": 1},  # 0 - Pause, 1 - Open, 2 - Close
+    "current_position": {"siid": 2, "piid": 2},  # Range: [0, 100, 1]
+    "target_position": {"siid": 2, "piid": 3},  # Range: [0, 100, 1]
+    "mode": {"siid": 2, "piid": 4},  # 0 - Normal,  1 - Reversal, 2 - Calibrate
+}
+
+# Model: "OnViz Curtain Controller (Wi-Fi)"
+MODEL_CURTAIN_BABAI = "babai.curtain.190812"
+
+
+class MotorControl(Enum):
+    Pause = 0
+    Open = 1
+    Close = 2
+
+
+class Mode(Enum):
+    Normal = 0
+    Reverse = 1
+    Calibrate = 2
+
+
+class CurtainStatus(DeviceStatus):
+    def __init__(self, data: Dict[str, Any]) -> None:
+        """Response from device.
+
+        {'motor_control': 0}
+        """
+        self.data = data
+
+    @property
+    def status(self) -> MotorControl:
+        """Device status."""
+        return MotorControl(self.data["motor_control"])
+
+    @property
+    def motor_control(self) -> MotorControl:
+        """Device status."""
+        return MotorControl(self.data["motor_control"])
+
```

Comment:
```suggestion
```
As this is just a duplicate of status, wouldn't it be better to remove this altogether?